src/freenet/node/Node.java

```diff
@@ -1312,13 +1320,37 @@ public Short get() {
 
 					@Override
 					public void set(Short val) throws InvalidConfigValueException {
-						if(maxHTL < 0) throw new InvalidConfigValueException("Impossible max HTL");
+						if(val < 0) throw new InvalidConfigValueException("Impossible max HTL");
```

Comment:
@xor-freenet I believe @nextgens and @Thynix were discussing it earlier and decided it wasn't worth the effort to separate it out. IIRC it was on IRC.